requirements.txt

```diff
@@ -0,0 +1,5 @@
+pylint
```

Comment:
do we need pylint and flake?

Reply:
I did a bit more digging and realized that for the most part pylint seem to be more thorough and verbose. The only thing it doesn't support out of the box is complexity check. I'm going to drop flake8.